content/hardware/04.pro/boards/portenta-h7/tutorials/over-the-air-update/content.md

```diff
@@ -138,6 +138,51 @@ Convert your encoded file into `.ota` format
 
 You can use `OTA_Usage_Portenta.ino.PORTENTA_H7_M7` as a sketch name for facilitated identification of the file. After this, you will have the `.ota` file of the sketch that you will use with the OTA process. 
 
+### Installing Python 3 On Ubuntu 
```

Comment:
```suggestion
### Installing Python 3 On Linux 
```

Hey, put Linux instead of Ubuntu. The tutorial is compatible with Linux and MAC so don't mention Ubuntu because there are different distros that can be used

**do the same in the rest of the document please**